docker-compose.override.yml

```diff
@@ -1,13 +1,16 @@
 version: '3.7'
 
+x-default-env: &x-default-env
+  RAILS_ENV: "${RAILS_ENV:-development}"
+  NODE_ENV: development
+  MAILSERVER_HOST: mailserver
+  MAILSERVER_PORT: 25
```

Comment:
@roschaefer I’m unsure if this is the best way to extend the environment in services. Maybe there is a Docker-native solution that does not rely on YAML anchors.

Reply:
We could have a look at this, but I’m not sure if it’ll work with the rest of the setup: https://docs.docker.com/compose/extends/#understand-the-extends-configuration